docs/development-guide.md

```diff
@@ -73,7 +73,19 @@ To customize / override a specific testing stage, please read the documentation
 4. [Flake8][flake8-docs]
 5. [Bandit][bandit-docs]
 
-### `setup.py`
+## Validate Examples Used in Documentation
+
+In the `docs/examples/` directory of this repo, there are example Python scripts which we use in our documentation.
+You can validate that the examples run properly using:
+
+```bash
+docker-compose run --rm validateDocExamples
+```
+
+When running, we expect the script to produce output.
```

Comment:
I feel like this sentence might be missing some content. What is the expected output? Or is the expectation actually "no output" on success.

Reply:
Thanks for the input @plannigan; I'll improve this.